fastlane/Fastfile

```diff
@@ -193,7 +193,7 @@ end
   # bundle exec fastlane new_hotfix_release version:10.6.1
   # bundle exec fastlane new_hotfix_release skip_confirm:true version:10.6.1
   #####################################################################################
-  desc "Creates a new hotfix branch from the given tag"
+  desc 'Creates a new hotfix branch for the given version:x.y.z. The branch will be cut from the tag x.y of the previous release'
```

Comment:
See my comment here: https://github.com/woocommerce/woocommerce-ios/pull/4703#discussion_r680648269

I'm mentioning it not because I feel strongly that it should be addressed, but just in case you look at one PR before the other.

Reply:
Noted. Sadly, some of the other PRs that are aligning the hotfix lanes in other repos have already been merged with that same description. With that and the fact that I'm not sure what could be a nice wording that explains this but also doesn't make the description too complex to understand trying to cover all cases, I'm not sure if it's worth re-opening yet another PR on the other repos just to fix that wording on the repos that have already had their hotfix-lane-PRs merged? wdyt?

Reply:
I agree with you. This can stay the way it is. 👍 

![ship it](https://media.giphy.com/media/79qf1N4RJtc8o/giphy.gif)